app/views/react.scala.html

```diff
@@ -6,5 +6,16 @@
 }
 
 @main(title = title, scripts = scripts) {
-    <main id="@id"></main>
+    <main id="@id">
+        <noscript>
+            <div style="text-align: center;
```

Comment:
Yes, I didn't want to mix this specific styles with the entire app.  I feel it is much more clear to have a hard separation between these styles and the ones that belong to the components.